sharding/nodesCoordinator/hashValidatorShuffler.go

```diff
@@ -82,8 +73,6 @@ func NewHashValidatorsShuffler(args *NodesShufflerArgs) (*randHashShuffler, erro
 	var configs []config.MaxNodesChangeConfig
 
 	log.Debug("hashValidatorShuffler: enable epoch for max nodes change", "epoch", args.MaxNodesEnableConfig)
```

Comment:
This debug is wrong. `args.MaxNodesEnableConfig` is a slice of configs. Not sure how debugger prints it.
If we want to keep this line and log.Debug prints it correctly, we can change it to:

```suggestion
	log.Debug("hashValidatorShuffler: enable epoch for max nodes change", "MaxNodesEnableConfig", args.MaxNodesEnableConfig)
```

Reply:
indeed, updated